scripts/create_short_url.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+set -o errexit -o pipefail
+cd "$(dirname "${BASH_SOURCE}")/.."
+
+if [[ $# -ne 2 && $# -ne 3 ]]; then
+    cat << EOF
+Usage: create_short_url.sh <target_link> <filename> [shortlink]
```

Comment:
Nit: If we ever rename the script this would be out of date, `$0` can be used instead of `create_short_url.sh`.